src/mumble/PipeWire.cpp

```diff
@@ -292,7 +292,7 @@ void PipeWireEngine::start() {
 }
 
 void PipeWireEngine::stop() {
-	if (m_loop) {
+	if (m_thread) {
```

Comment:
What did the previous condition check for and why is it safe to remove it instead of checking both conditions?

Reply:
The previous condition was probably a typo, as there is no reason to assume `m_thread` exists when `m_loop` does. It's actually the opposite, with `m_thread` depending on `m_loop`:

https://github.com/mumble-voip/mumble/blob/31127868626c186327bc5e5e0a5eec61ecc6860e/src/mumble/PipeWire.cpp#L190-L211